src/block/carousel/frontend-carousel.js

```diff
@@ -93,11 +93,15 @@ class _StackableCarousel {
 					const clone = original.cloneNode( true )
 					clone.classList.add( `stk-slide-clone-${ slideIndex + 1 }` )
 					clone.style.zIndex = -1
+
+					clone.style.willChange = 'transform'
+					clone.style.transform = 'TranslateX( 0 )'
```

Comment:
The capitalized form might not be accepted by all browsers:
```suggestion
					clone.style.transform = 'translateX( 0 )'
```